instrumentation/opentelemetry-instrumentation-flask/src/opentelemetry/instrumentation/flask/__init__.py

```diff
@@ -327,7 +359,22 @@ def __init__(self, *args, **kwargs):
             __name__, __version__, _InstrumentedFlask._tracer_provider
         )
 
+        meter = get_meter(
+            __name__, __version__, _InstrumentedFlask._meter_provider
+        )
+        duration_histogram = meter.create_histogram(
+            name="http.server.duration",
+            unit="ms",
+            description="measures the duration of the inbound HTTP request",
+        )
+        active_requests_counter = meter.create_up_down_counter(
```

Comment:
If we have separate metric instruments for if user uses `_instrument` and `instrument_app`, does that mean that they will get different metric streams representing "metrics for all flask apps" and "metrics for a certain flask app"?